lib/github_changelog_generator/octo_fetcher.rb

```diff
@@ -442,6 +452,7 @@ def iterate_pages(client, method, *arguments, parent: nil, **options)
     # This is wrapper with rescue block
     #
     # @return [Object] returns exactly the same, what you put in the block, but wrap it with begin-rescue block
+    # @param [Proc] block
```

Comment:
Yard probably has some notation for you here, specific to blocks. We don't want the docs to show that this method takes a positional argument. Does the rendered Yard docs HTML show that?